.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.3', '1.5']
+        julia-version: ['1.6']
```

Comment:
I would still test in 1.3 (we can exclude the doctests in this case). I just pushed the change. If this works, we can merge.